src/js/util.js

```diff
@@ -218,6 +284,12 @@ var Util;
             return false;
         },
 
+        findParentBlockContainer: function (node) {
```

Comment:
No, because we want to find the closest block container to the element, not the top container. I'll change the name to `Util.getClosestBlockContainer` to make the difference between these two methods clear.

For instance in the following html:

    <div id=medium-editor>
    <blockquote>
        <pre>Here is some pre-formatted text</pre>
        <p>And here is some non preformatted text</p>
    </blockquote>
    </div>

If the user has selected such that `startContainer` is inside the `pre` tag and `endContainer` inside the `p` tag (e.g. a selection of "pre-formatted text And here is some", we want to know this and fall back to the browser `createLink` command rather than attempting to split up tags, which would result in changing the format of the text (due to duplicating `pre` or `p` tags which have implicit line break semantics that the user won't want us to make).